rudder-web/src/main/scala/com/normation/rudder/web/snippet/HomePage.scala

```diff
@@ -369,15 +369,15 @@ class HomePage extends Loggable {
       agentSoftDn      =  agentSoftEntries.map(_.dn.toString).toSet
 
       agentSoft        <- sequence(agentSoftEntries){ entry =>
-                            (mapper.softwareFromEntry(entry) ?~! "Error when mapping LDAP entry %s to a software".format(entry)).map { s =>
+                            (mapper.softwareFromEntry(entry) ?~! s"Error when mapping LDAP entry ${entry} to a software").map { s =>
                               //here, we want to use Agent Version display name, not the software one
                               s.name match {
                                 case None => s
                                 case Some(name) => name.toLowerCase match {
-                                  case NOVA_AGENT.inventorySoftwareName =>
-                                    s.copy(version = s.version.map(v => new Version(NOVA_AGENT.toAgentVersionName(v.value))))
-                                  case ag if ag == DSC_AGENT.inventorySoftwareName.toLowerCase =>
-                                    s.copy(version = s.version.map(v => new Version(DSC_AGENT.toAgentVersionName(v.value))))
+                                  case CfeEnterprise.inventorySoftwareName =>
+                                    s.copy(version = s.version.map(v => new Version(CfeEnterprise.toAgentVersionName(v.value))))
+                                  case ag if ag == Dsc.inventorySoftwareName.toLowerCase =>
```

Comment:
Different for CfeEnteprise and dsc because we need to lowercase the matching value and could not use it directly (You can't do "case  Dsc.inventorySoftwareName.toLowerCase " compiler prevents it